hedera-node/hedera-evm/build.gradle.kts

```diff
@@ -31,11 +29,11 @@ testModuleInfo {
     requires("org.mockito.junit.jupiter")
 }
 
-publishing {
-    publications {
-        named<MavenPublication>("maven") {
-            groupId = "com.hedera.evm"
-            artifactId = "hedera-evm"
-        }
-    }
+// Second publication using the 'group' and 'name' under wich 'evm'
+// was published before
+publishing.publications.create<MavenPublication>("mavenLegacy") {
+    from(components["java"])
+
+    groupId = "com.hedera.evm"
+    artifactId = "hedera-evm"
```

Comment:
❓ If we change the coordinates, other services components that we now publish would break as they assume the unfied coordinates in their dependencies. The way to keep things working is too publish this component twice with the new and "old" coordinates. Is this fine? Maybe this second publication can then be dropped late (1.0 release?).

Reply:
@jjohannes We must be able to support publishing different components into different maven central group IDs. Publishing into two different coordinates is not an acceptable solution. The EVM should be isolated into the `com.hedera.evm` Group ID and all other components should be referencing the EVM via the `com.hedera.evm` Group ID.

Reply:
@jjohannes I have applied a set of changes which maintain the EVM MC coordinates. This is necessary as well because other projects depend on and use these coordinates.